### PR TITLE
Add header to new landing page

### PR DESCRIPTION
**Pull request by Viacheslav-Chopyk**
add header

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,22 +1,49 @@
...
+  <div class="w-full max-w-[1440px]">
+    <nav class="bg-transparent w-full z-20 text-black flex justify-between items-center h-[90px] max-w-[1200px] mx-auto mb-[57px]">
+      <div class="h-[70px] justify-between items-center w-full flex">
+        <div class="flex items-center">
+          <a href="#">
+            <img src="assets/new_logo.svg" alt="Bee O`clock Logo">
+          </a>
+        </div>
+        <ul class="hidden md:flex gap-[60px]">
+          <li><a href="#" class="hover:text-yellow-500">Послуги</a></li>
```
[the remark this answers is not in the document]
Comment (Viacheslav-Chopyk) [reply src/app/app.component.html:11]: The translation will be later

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,22 +1,49 @@
...
+          </div>
+          <button type="button" class="font-[FixelDisplay-Bold] text-[20px] py-4 px-7 bg-[#FFD429] rounded-[60px]">
+            Спробувати безкоштовно
+          </button>
+        </div>
+        <div class="relative">
+          <img ngSrc="assets/img/main_img.png" alt="" height="753" width="721">
+          <img class="absolute top-[30%] right-[90%]" ngSrc="assets/img/vector_1.png" alt="" height="78" width="40">
+          <img class="absolute top-[67%]" ngSrc="assets/img/vector_2.png" alt="" height="48" width="78">
+          <img class="absolute top-[4%] right-[67%]" src="assets/img/service_details.png" alt="">
```
[the remark this answers is not in the document]
Comment (Viacheslav-Chopyk) [reply src/app/app.component.html:41]: done

File: src/app/app.component.scss [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
-  background-color: #000;
-  height: 90px;
-  display: flex;
-  align-items: center;
-  justify-content: center;
+.wrapper {
+  nav {
+    ul li a {
+      font-family: 'FixelDisplay-Medium', sans-serif;
+      font-size: 15px;
```
[the remark this answers is not in the document]
Comment (Viacheslav-Chopyk) [reply src/app/app.component.scss:5]: done

File: src/common/cdk/social-share.seo.service.ts [hunk 4]
```diff
@@ -345,15 +345,15 @@ export class SocialShareSeoService {
     const canonicalElement = this.document.head.querySelector(selector);
     if (canonicalElement) {
       // this.document.head.removeChild(canonicalElement);
-      this.renderer2.removeChild(this.document.head, canonicalElement);
+      /*this.renderer2.removeChild(this.document.head, canonicalElement);*/
```
[the remark this answers is not in the document]
Comment (Karbashevskyi) [reply src/common/cdk/social-share.seo.service.ts:348]: @Bozonych 

I said do it, it is specific case which I have to fix later, so we can leave it here and other comment which relates with renderer2, document and SocialShareService

File: src/styles.scss [hunk 5]
```diff
@@ -3,3 +3,24 @@
 @tailwind utilities;
 
 @import "bootstrap-icons";
+
+@font-face {
+  font-family: 'FixelDisplay-Bold';
+  src: url('/assets/fonts/FixelDisplay-Bold.woff2') format('woff2');
```
Comment (Karbashevskyi) [inline src/styles.scss:9]: replace the line with: `src: url('./assets/fonts/FixelDisplay-Bold.woff2') format('woff2');`

File: src/styles.scss [hunk 6]
```diff
@@ -3,3 +3,24 @@
...
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'FixelDisplay-Medium';
+  src: url('/assets/fonts/FixelDisplay-Medium.woff2') format('woff2');
```
Comment (Karbashevskyi) [inline src/styles.scss:16]: replace the line with: `src: url('./assets/fonts/FixelDisplay-Medium.woff2') format('woff2');`

File: src/styles.scss [hunk 7]
```diff
@@ -3,3 +3,24 @@
...
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'FixelDisplay-Regular';
+  src: url('/assets/fonts/FixelDisplay-Regular.woff2') format('woff2');
```
Comment (Karbashevskyi) [inline src/styles.scss:23]: replace the line with:  `src: url('./assets/fonts/FixelDisplay-Regular.woff2') format('woff2');`

Comment (Karbashevskyi) [reply src/styles.scss:23]: @Viacheslav-Chopyk

File: src/styles.scss (hunk 6 above)
Comment (Karbashevskyi) [reply src/styles.scss:16]: @Viacheslav-Chopyk

File: src/styles.scss (hunk 5 above)
Comment (Karbashevskyi) [reply src/styles.scss:9]: @Viacheslav-Chopyk

File: src/app/app.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (Viacheslav-Chopyk) [reply src/app/app.component.html]: done

File: src/styles.scss (hunk 7 above)
Comment (Viacheslav-Chopyk) [reply src/styles.scss]: done

File: src/styles.scss (hunk 6 above)
Comment (Viacheslav-Chopyk) [reply src/styles.scss]: done

File: src/styles.scss (hunk 5 above)
Comment (Viacheslav-Chopyk) [reply src/styles.scss]: done

**Pull request merged**